client/state/posts/utils.js

```diff
@@ -31,7 +31,9 @@ import stripHtml from 'lib/post-normalizer/rule-strip-html';
  * Constants
  */
 
-const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/;
+const REGEXP_SERIALIZED_QUERY = /^((\d+):)?(.*)$/,
+	REGEXP_EMPTY_CONTENT = /^<p>(<br[^>]*>|&nbsp;|\s)*<\/p>$/,
+	CONTENT_LENGTH_ASSUME_SET = 50;
```

Comment:
We previously had a guideline _against_ one-per-line variables, which was removed in #4251. There's currently no rule one way or the other. I've started leaning more toward one-per-line, since the previous guideline was a result of needing to group all `var`s at the top of a function. As that's no longer necessary and we expect many `const` and `let` throughout a function anyways, grouping them into a comma-separate set seems to have no real benefit. To a lesser degree, tabbing on lines after `const` is ugly 😄